src/ispc.h

```diff
@@ -621,6 +623,9 @@ struct Globals {
 
     /** Lines for which warnings are turned off. */
     std::map<std::pair<int, std::string>, bool> turnOffWarnings;
+
+    /* If true, we are compiling for more than one target. */
+    bool isMultiTargetCompilation;
```

Comment:
Why is this a flag, but not a function? `targets.size() > 1` check doesn't seem to be too expensive.

Reply:
We don't really save the targets anywhere. It's passed into 'Module::CompileAndOutput' and we make decisions for targets individually. So we either store all targets somewhere or just set a flag when we know it's multi-target.

Reply:
Ok, makes sense.